src/libsyncengine/jobs/network/csvfullfilelistwithcursorjob.cpp

```diff
@@ -103,14 +203,14 @@ bool CsvFullFileListWithCursorJob::getItem(SnapshotItem &item, bool &error, bool
             return true;
         }
 
-        // File name could have line return in it. If so, read next line and continue parsing
+        // A file name surrounded by double quotes can have a line return in it. If so, read next line and continue parsing
         if (readingDoubleQuotedValue) {
             tmp.push_back('\n');
             readNextLine = true;
-            std::getline(_ss, line);
+            std::getline(ss, line);
             if (line.empty()) {
                 LOGW_WARN(_logger, L"Invalid line");
-                ignore = true;
+                error = true;
```

Comment:
This is the fix.

Reply:
Do you think you can display an explicit messgae to the user explaining the problem. With perhaps the file name (or at least a part of it) the causing the parsing issue.

Reply:
Done in the next commit.

Reply:
Sorry, what I meant was adding an error in the error list explaining why the sync stopped explicitly. Probably calling the `SyncPal::addError` method somewhere.
Do you think it is possible?